cypress/e2e/course_board/createEditDeleteColumnInBoard.feature

```diff
@@ -1,33 +1,53 @@
-    Given I am logged in as a 'teacher1_nbc' at 'nbc'
+    # pre-condition: teacher and student log in to create their account in a same school
+    Given I am logged in as a '<teacher>' at '<namespace>'
+    Given I am logged in as a '<admin>' at '<namespace>'
+
+    # pre-condition: admin enables student visiblity for a teacher to create a new course
+    #When I go to administration page
+    #When I go to school administration
+    # Note: remove the following line if old admin page is hidden
+    # Note: Currently admin page is not opening due to the issue on admin user created by using school API.
+    #When I go to new school administration page
+    #When I click on general settings panel
+    #When I click the toggle switch to enable student visibility for teachers
+    #When I click on button Save admin settings
+
+    # pre-condition: admin creates a course and assign teacher and student to the course
     When I go to rooms overview
-    When I click on FAB to add or import courses
     When I click on FAB to create a new room
-    When I enter the course title 'Cypress Test Creation and Deletion'
+    When I click on new course create button in sub menu
+    Then I see section one area on the course create page
+    When I enter the course title 'CypressAut Test Creation and Deletion'
     When I select room colour as red
-    Then I see teacher 'Karl Herzog' is selected by default
+    Then I select teacher '<fullname_teacher>' is selected by default
     Then I see substitute teacher selection box
     Then I see date pickers to start and end the course as per school year
     Then I see button to create a course time table container
     When I click on button Next Steps after entering the room detail in section one
     Then I see section two area on the course create page
     Then I see class selection box to select the class for the room
-    Then I see student selection box to select the class for the room
+    Then I see student selection box to select the student for the room
+    # Note: Student user is not needed in this feature so this step is commented out
+    #When I select the student '<fullname_student>' in the list
     When I click on button Next Steps after selecting room participant details
-    Then I see the section three area as the finish page
+    Then I see the section three as the finish page
     When I click on button To Course Overview on the finish page
-    Then I see the course 'Cypress Test Creation and Deletion' on the room overview page
+    # Note: This step is not applicable for the admin user
+    #Then I see the course 'CypressAut Test Creation and Deletion' on the room overview page
 
-    # Teacher adds a new Board as a pre-condition ---
-    When I go to room 'Cypress Test Creation and Deletion'
+    # teacher adds a new Board
+    Given I am logged in as a 'teacher1_brb' at 'brb'
+    When I go to rooms overview
+    When I go to room 'CypressAut Test Creation and Deletion'
```

Comment:
Same comment as mentioned above. Check in all places as I don't want to comment in every place like title names (course, topic, task, board, column, card) etc

Reply:
what do you mean? using example table variable?

Reply:
Tho this PR is mainly for API data related adaptation. But I updated it for this feature file.